auth-table-based-service/src/test/java/io/stargate/auth/AuthenticationSubjectTest.java

```diff
@@ -93,4 +93,14 @@ public void fullInputsExternalWithNoCustomProperties() {
     assertThat(subject.isFromExternalAuth()).isEqualTo(true);
     assertThat(subject.customProperties()).isEmpty();
   }
+
+  @Test
+  public void asUser() {
```

Comment:
can we change the name of this test to be more meaningful?

Reply:
Renamed to `convertToAuthenticatedUser`, hope it's more readable :)